### ISSUE TITLE
tsconfck forces pnpm consumers to update pnpm

### ISSUE THREAD

**Issue by dyantako** [dominikg/tsconfck#78]
## Description

We are using an older version of pnpm (`7.13.3`) and cannot install the `tsconfck` `2.0.2`. The package.json for this release has dropped support for older versions of pnpm, and we get an error because we have `engine-strict=true` set in our npmrc.

```
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

This error happened while installing the dependencies of @ladle/react@2.9.0
 at vite-tsconfig-paths@4.0.5

Your pnpm version is incompatible with "registry.npmjs.org/tsconfck/2.0.2".

Expected version: ^7.18.0
Got: 7.13.3

This is happening because the package's manifest has an engines.pnpm field specified.
To fix this issue, install the required pnpm version globally.
```

In the [package.json](https://github.com/dominikg/tsconfck/blob/main/package.json#L108), `engines.pnpm` is set to `^7.18.0` for this release, and `^7.26.0` in main. It looks like it's being updated by renovate to keep up to the latest version. I think this should specify a more general supported version e.g. `>=7`.

**Comment by dominikg** [comment]
released in 2.0.3

**Comment by dyantako** [comment]
Fantastic, thank you 🙇
